CMakeLists.txt

```diff
@@ -7,9 +7,11 @@
 #        All rights reserved.
 #------------------------------------------------------------------------------#
 
-# Build notes: http://rtt/redmine/projects/draco/wiki/Configuration_and_Building
+# Build notes: 
+#   https://rtt.lanl.gov/redmine/projects/draco/wiki/
+#   https://rtt.lanl.gov/redmine/projects/draco/wiki/Common_Configure_Options
 
-cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.9.0 FATAL_ERROR)
```

Comment:
This is a WIP because we must move to cmake-3.9.0+ on all machines before it can be merged.  I wanted to capture the fix while I was thinking about it.  Unfortunately, I haven't been able to make this change work for cmake version < 3.9. I will leave the commented code in `compilerEnv.cmake` for now to remind me of the necessary changes.